OpenCover.UI.TestDiscoverer/NUnit/NUnitDiscoverer.cs

```diff
@@ -111,7 +135,11 @@ private TestMethod[] DiscoverTestsInClass(TypeDefinition type, TestClass @class)
 			foreach (var method in type.Methods)
 			{
 				bool isTestMethod = false;
-				var trait = new List<string>();
+                var traits = new List<string>();
+                if (@class.Traits != null && @class.Traits.Length > 0)
+                {
+                    traits.AddRange(@class.Traits);
+                }
```

Comment:
Do you have the NUnit test runner installed in Visual Studio? It's an extension to execute (and debug) NUnit tests from inside Visual Studio. You can install it in Visual Studio using the menu: Tools > Extensions and Updates ... > and search for "NUnit test adapter". It should show the tests in the Visual Studio Test Explorer:

![image](https://cloud.githubusercontent.com/assets/1618378/19413072/251c3a8a-9324-11e6-9968-9de0244a7fdb.png)

Since you implemented this for NUnit tests, I think you might already have it installed?